receiver/awscontainerinsightreceiver/internal/k8sapiserver/prometheus_scraper.go

```diff
@@ -33,24 +33,25 @@ const (
 var (
 	controlPlaneMetricAllowList = []string{
 		"apiserver_admission_controller_admission_duration_seconds.*",
```

Comment:
Do we not want a `^` at the start of this?

Reply:
yeah good eye